label_sync/labels.yaml

```diff
@@ -1543,3 +1543,10 @@ repos:
         name: kind/user-story
         target: issues
         addedBy: humans
+  kubernetes-sigs/kind:
+    labels:
+      - color: 970fff
+        description: Issues or PRs related to podman
+        name: area/provider/podman
```

Comment:
we should add both?

Reply:
I though about that, but this is very specific and the other is a catch all.
are we going to use the docker label it at the end of the day ?

Reply:
Is "provider" the right term here?  I see it overloaded/used in so many contexts throughout the project (e.g. cluster provider, cloud provider, credentials provide, test/e2e/frameworks' "provider).  Is there a more specific term that would make more sense?

Reply:
thing is that in this case it matches perfectly :smile: 
https://github.com/kubernetes-sigs/kind/tree/master/pkg/cluster/internal/providers/podman

Reply:
> are we going to use the docker label it at the end of the day ?

yes, when there are docker provider specific bugs

Reply:
What about aaron comment? do we want another term instead of provider? I'm very bad a naming so I will copy it verbatim

Reply:
"node provider" is the full term but I don't think that's helpful here.
it's OK because area/ is repo specific normally anyhow